### PR TITLE
Notes for the IPFS Community Call 2019-1-28

### PR DESCRIPTION
**Pull request by pkafei**

### CODE REVIEW

File: meeting-notes/2019/Q1/2019-1-28--ipfs-community-wg-weekly.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  
+  
+
+
+**Main Topic:**The IPFS Pinata Project
+
+**Presenter:** Matt Ober
+
+**Video:**
```
[the remark this answers is not in the document]
Comment (pkafei) [reply meeting-notes/2019/Q1/2019-1-28--ipfs-community-wg-weekly.md]: @daviddias Good catch, I'll fix the link.

[the remark this answers is not in the document]
Comment (daviddias) [reply meeting-notes/2019/Q1/2019-1-28--ipfs-community-wg-weekly.md]: #protip - always review your own PRs. I catch a lot of my hiccups reviewing my PRs as if I was reviewing someone elses :)

[the remark this answers is not in the document]
Comment (pkafei) [reply meeting-notes/2019/Q1/2019-1-28--ipfs-community-wg-weekly.md]: I have not been automatically saving chats- I'll just simply go over the video again and capture the questions. 

In the future, for anyone who wants to automatically save chats you can do this by going under _Meeting Settings/In Meeting (Basic) and then turn on _Auto saving chats_ https://support.zoom.us/hc/en-us/articles/115004792763-Saving-In-Meeting-Chat

[the remark this answers is not in the document]
Comment (daviddias) [reply meeting-notes/2019/Q1/2019-1-28--ipfs-community-wg-weekly.md]: @pkafei oh I even thought that was a default. This seems like a good thing to have on the [Tutorial on "How to host a call"](https://github.com/ipfs/team-mgmt/blob/master/HOST_A_CALL.md). Can you add that note there?

**Pull request merged**